### DISCUSSION TITLE
✨ Netbird

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2227] [Requests]
### Description

NetBird combines a configuration-free peer-to-peer private network and a centralized access control system in a single platform, making it easy to create secure private networks for your organization or home.

Connect. NetBird creates a WireGuard-based overlay network that automatically connects your machines over an encrypted tunnel, leaving behind the hassle of opening ports, complex firewall rules, VPN gateways, and so forth.

Secure. NetBird enables secure remote access by applying granular access policies, while allowing you to manage them intuitively from a single place. Works universally on any infrastructure.

### Official Website

https://netbird.io/

### Documentation link

https://netbird.io/docs/

### Last application release & date

0.25.4 (16/01/2024)

### Application license

BSD-3-Clause license

### Source code repository link

https://github.com/netbirdio/netbird

### Docker image link

https://hub.docker.com/r/netbirdio/netbird

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote 👍 this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
Originally requested by @RobotsAreCrazy

**Comment by OriginalSimon** [comment]
Looking forward to seeing it in the app store.
